types/jest/index.d.ts

```diff
@@ -312,6 +331,40 @@ declare namespace jest {
         virtual?: boolean | undefined;
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    type MockableFunction = (...args: any[]) => any;
+    type MethodKeysOf<T> = { [K in keyof T]: T[K] extends MockableFunction ? K : never }[keyof T];
+    type PropertyKeysOf<T> = { [K in keyof T]: T[K] extends MockableFunction ? never : K }[keyof T];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    type ArgumentsOf<T> = T extends (...args: infer A) => any ? A : never;
```

Comment:
this can be replaced with the built-in `Parameter` type

Reply:
why `ts-jest` not implements these built-in types? I'm currently on mobile (out of station) can u please push these changes?

Reply:
I'm resolving this conversation